auth-server/src/main/java/com/antonio/authserver/dto/RealmRelationsDto.java

```diff
@@ -0,0 +1,30 @@
+package com.antonio.authserver.dto;
```

Comment:
unde aveti nevoie de toate informatiile in acelasi timp?

Reply:
@radasilviu are nevoie @SerbanGabriel ca sa nu faca 300 calluri pt 1 buton

Reply:
Aici salvez RealmRelationDto in ReplaySubject si manipulez datele de acolo,sa nu mai fac call pentru a lua users,clients,roles,userRoles.

Reply:
Din ce inteleg eu, tu aduci toate datele la selectarea unui realm ?